script.js

```diff
@@ -0,0 +1,12 @@
+var text = 'Velociraptor is a genus of herbivorous ceratopsid dinosaur that first appeared during the late Maastrichtian stage of the late Cretaceous period.';
+var dinosaur = 'triceratops';
+
+var setUpperCase = dinosaur.toUpperCase();
```

Comment:
name of this variable suggest that this is a method :) better will be name e.g. `upperCasedName`